server/project.go

```diff
@@ -636,10 +636,11 @@ func (s *Server) setupVCSSQLReviewCI(ctx context.Context, repository *api.Reposi
 		repository.VCS.InstanceURL,
 		repository.ExternalID,
 		&vcsPlugin.PullRequestCreate{
-			Title: sqlReviewInVCSPRTitle,
-			Body:  "This pull request is auto-generated by Bytebase.",
-			Head:  branch.Name,
-			Base:  repository.BranchFilter,
+			Title:                 sqlReviewInVCSPRTitle,
+			Body:                  "This pull request is auto-generated by Bytebase for VCS workflow.",
```

Comment:
In code we call VCS workflow, but in product we've been using "GitOps workflow", I think we should use "GitOps workflow" here.

Reply:
Sure. Updated